src/Nethermind/Nethermind.Crypto/TransactionExtensions.cs

```diff
@@ -4,17 +4,17 @@
 using Nethermind.Core;
 using Nethermind.Core.Crypto;
 using Nethermind.Serialization.Rlp;
+using System;
 
 namespace Nethermind.Crypto
 {
     public static class TransactionExtensions
     {
-        private static readonly TxDecoder _txDecoder = TxDecoder.Instance;
```

Comment:
Couldn't it just be a property instead?
```
private static TxDecoder _txDecoder => TxDecoder.Instance;
```
Or even not exist and just use `TxDecoder.Instance` directly?

Reply:
I did it in such way, but it looked not very performant